### ISSUE TITLE
[bug]: pushing the first image to github registry from local machine breaks the pipelines

### ISSUE THREAD

**Issue by LucaLanziani** [nearform/initium-cli#149]
When pushing an image from local using something like:

```
initium onbranch --stop-on-push
```

The resulting package will not be associated with the repo causing the pipelines to fail with:
```
Failed with denied: permission_denied: write_package
```

This behaviour could be fixed associating the repository with the package using the appropriate label https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry#labelling-container-images

We have to test this option and if it works add the label to the docker files if we are pushing to github.

**Comment by mablanco** [comment]
However, Github recognises a properly labeled image pushed from the CLI as a package belonging to the repo.

**Comment by mablanco** [comment]
Even then, the workflow keeps on failing (see https://github.com/mablanco/initium-nodejs-demo-app/actions/runs/6955880255/job/18925574420?pr=3#step:4:210).

**Comment by mablanco** [comment]
I've created a new repository and initialized it with Initium. After trying with a couple of different branches and PRs, all tests were successful and each pipeline was able to push the Docker image it built.
![image](https://github.com/nearform/initium-cli/assets/7761778/8a618c79-a3c3-4f59-bbcc-95afff2b159a)

**Comment by mablanco** [comment]
So I started comparing the package manager of both repositories and I got the following main difference.

**Failing repo**
![image](https://github.com/nearform/initium-cli/assets/7761778/89210696-b2a1-4668-80aa-7ab407efefe8)

**Working repo**
![image](https://github.com/nearform/initium-cli/assets/7761778/aa443925-188a-4841-a0f5-aaa5fc126b97)

As can be seen in the above screenshots, both package managers are already assigning the proper org.opencontainers label to the pushed images. However, the failing one lacks privileges for repository access, while the working one has Admin role privileges assigned.

So my theory is that this issue is happening in repositories that may have been modified or have been created with an old version of Initium, as the latest one doesn't show the problem. I'm going to manually assign the Write role for repository access in the failing repo and check if the pipelines work again.

**Comment by mablanco** [comment]
Yes, it has worked! (see https://github.com/mablanco/initium-nodejs-demo-app/actions/runs/6955880255/job/18926680865#step:4:232).

I'm going to close the PR and mode to ticket to **Waiting for review** column.

**Comment by dublx** [comment]
Lots of new things for me to learn on GitHub but your comment makes sense.
